packages/core/src/utils.ts

```diff
@@ -161,7 +161,6 @@ npx hardhat chugsplash-deploy --network ${networkName} ${configPath}
   }
 
   if (requiresOwnershipTransfer.length > 0) {
-    // TODO update this once the transfer ownership task is implemented
     throw new Error(
       `Error: Detected proxy contracts which are not managed by ChugSplash.
       ${requiresOwnershipTransfer.map(
```

Comment:
Could you fill out the task on line 171 of this file? It's currently `npx hardhat chugsplash-transfer-ownership>`. Something like:
```
npx hardhat chugsplash-transfer-ownership --network ${hre.network.name} --proxy-address <address> ${configPath}
```